utils/PathSanitizingFileCheck

```diff
@@ -73,16 +79,20 @@ constants.""")
         # We are replacing the Unix path separators in the paths passed as
         # arguments with a broader pattern to also allow forward slashes and
         # double escaped slashes in the result that we are checking. Sigh.
-        stdin = re.sub(re.sub(r'/', slashes_re, pattern), replacement, stdin)
-
-    p = subprocess.Popen(
-        [args.file_check_path] + unknown_args, stdin=subprocess.PIPE)
-    stdout, stderr = p.communicate(stdin)
-    if stdout is not None:
-        print(stdout)
-    if stderr is not None:
-        print(stderr, file=sys.stderr)
-    return p.wait()
+        stdin = stdin.replace(re.sub(r'/', slashes_re, pattern), replacement)
```

Comment:
I think the following will work for everyone. Can you check in your setup?

```
# Since we want to use pattern as a regex in some platforms, we need to
# escape it first, and then replace the escaped slash literal (r'\\/') for our
# platform-dependent slash regex.
stdin = re.sub(re.sub(r'\\/', slashes_re, re.escape(pattern)), replacement, stdin)
```

Reply:
Looks like it works, thank you!